examples/diffie_hellman_key_exchange.rs

```diff
@@ -38,11 +32,11 @@ where
 fn main() {
     let curve_name = std::env::args().nth(1);
     match curve_name.as_deref() {
-        Some("secp256k1") => ecdh::<curv::elliptic::curves::secp256_k1::GE>(),
-        Some("ristretto") => ecdh::<curv::elliptic::curves::curve_ristretto::GE>(),
-        Some("ed25519") => ecdh::<curv::elliptic::curves::ed25519::GE>(),
-        Some("bls12_381") => ecdh::<curv::elliptic::curves::bls12_381::g1::GE>(),
-        Some("p256") => ecdh::<curv::elliptic::curves::p256::GE>(),
+        Some("secp256k1") => ecdh::<Secp256k1>(),
+        // Some("ristretto") => ecdh::<curv::elliptic::curves::curve_ristretto::GE>(),
+        // Some("ed25519") => ecdh::<curv::elliptic::curves::ed25519::GE>(),
+        // Some("bls12_381") => ecdh::<curv::elliptic::curves::bls12_381::g1::GE>(),
+        // Some("p256") => ecdh::<curv::elliptic::curves::p256::GE>(),
```

Comment:
why these are commented out/not updated ?

Reply:
Ah, forgot to update, thanks!

Reply:
Fixed